docs/pages/1_user/02_connect_me.md

```diff
@@ -53,67 +50,78 @@ The following steps describe how to download and install the Shieldoo desktop cl
 
-To see the list of available servers, click on the client status icon in the system tray ![](../../images/ConnectMe13.png) and choose __My Access Rights In Shieldoo network__.
+2. Click on the __Sign-in__ option and log in with the same credentials you used to [log in to the Shieldoo web application](/connect_me/#login-to-shieldoo-web-application).
 
-{% include img.html src="../../images/ConnectMe14.png" alt="My Access Rights In Shieldoo network" %}
+    {% include img.html src="../../images/ConnectMe20.png" alt="Connect Me | Download & Install" %}
 
-The other possibility is to go to __Connect Me__ \| __My Access Rights__ in Shieldoo web application.
+3. Click on the __Connect__ option and connect to the network.
+
+    {% include img.html src="../../images/ConnectMe21.png" alt="Connect Me | Download & Install" %}
+
+After successfully connecting to the Shieldoo Secure Network, you can now securely connect to any server and attached services in the network.
+
+
+{: .note }
+>The __Shieldoo client icon in the system tray__ provides visual aid about its current status:
+>- Not Sign-in and Not Connect
+> ![](../../images/ConnectMe10.png)
+>- Sign-in and Not Connect
+> ![](../../images/ConnectMe11.png)
+>- Sign-in and Connect
+> ![](../../images/ConnectMe12.png)
+
+{: .note }
+>The __Shieldoo client includes additional options__:
+>- Update Shieldoo Client - _If a new client version is available, you can update it using this option_.
+>- Disconnect - _Use this option to disconnect from the network without losing your login credentials_.
+>- Specify Shieldoo network - _Use this option to change the network URL parameter_.
+>- Favorite Shieldoo Networks - _Use this option to save a list of your networks and switch between them._
+>- Enable Auto Start - _The use option allows the client to start automatically when the OS boots._ 
+>- Enabled automatic disconnect - _The use option allows you to save costs because, after 20 minutes of inactivity to connect to any server or access service, the client will automatically disconnect from the network._
+>- My access rights on the Shieldoo network - _Use this option to redirect to a list of my access rights._
+>- Version - _Displays the currently installed version of the client._
+>- Exit - _Use option to exit the client._
+
+### Connect to Servers
+To see the list of available servers, go to __Connect Me__ \| __My Access Rights__ in the Shieldoo web application.
 
 {% include img.html src="../../images/ConnectMe15.gif" alt="Connect Me | My Access Rights" %}
 
-It will show the list of available servers within your Shieldoo Secure Network with information about their names, IP addresses and description. In descriptions, your administrator will tell you how to connect to the server.
+It will show the list of available servers with information about their names, IP addresses and descriptions. In the descriptions, your admin will tell you what a particular server is for and how to connect to it.
```

Comment:
```suggestion
It will show the list of available servers with information about their names, IP addresses and descriptions. In the description, your admin tells you what a particular server is for and how to connect to it.
```